implementations/micrometer-registry-statsd/src/main/java/io/micrometer/statsd/StatsdMeterRegistry.java

```diff
@@ -15,8 +15,19 @@
  */
 package io.micrometer.statsd;
 
-import io.micrometer.core.instrument.*;
+import io.micrometer.core.instrument.Clock;
```

Comment:
My IntelliJ settings do this automatically, but if you prefer the wildcards, I will undo this.

Reply:
We generally prefer non `*` imports.

Reply:
Ah, not in Micrometer @spencergibb. It's the IntelliJ default, and they are everywhere ;)